src/iidm/NetworkIndex.cpp

```diff
@@ -81,11 +85,28 @@ stdcxx::range<MultiVariantObject> NetworkIndex::getAll<Identifiable, MultiVarian
     return boost::adaptors::values(m_objectsById) | boost::adaptors::indirected | boost::adaptors::filtered(filter) | boost::adaptors::transformed(mapper);
 }
 
+template <>
+stdcxx::const_range<VoltageLevel> NetworkIndex::getAll<VoltageLevel, VoltageLevel>() const {
+    stdcxx::const_range<VoltageLevel> bbvl = getAll<BusBreakerVoltageLevel, VoltageLevel>();
+    return boost::range::join(bbvl, getAll<NodeBreakerVoltageLevel, VoltageLevel>());
+}
+
+template <>
+stdcxx::range<VoltageLevel> NetworkIndex::getAll<VoltageLevel, VoltageLevel>() {
+    stdcxx::range<VoltageLevel> bbvl = getAll<BusBreakerVoltageLevel, VoltageLevel>();
+    return boost::range::join(bbvl, getAll<NodeBreakerVoltageLevel, VoltageLevel>());
+}
```

Comment:
Two options: create two local variables or none, but do not implement these methods like this.